ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/WidgetOptionsResetButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { MouseEventHandler } from 'react';
+import { Button, Tooltip } from '@patternfly/react-core';
+
+import { RedoIcon } from '@patternfly/react-icons';
+
+export type WidgetOptionsResetButtonProps = {
+    onClick: MouseEventHandler<HTMLButtonElement>;
+};
+
+const buttonLabel = 'Revert to default options';
+
+function WidgetOptionsResetButton({ onClick }: WidgetOptionsResetButtonProps) {
+    return (
+        <Tooltip content={buttonLabel}>
+            <Button
+                aria-label={buttonLabel}
+                variant="plain"
+                icon={<RedoIcon />}
```

Comment:
The "refresh" semantics of this icon concerns me. I searched for different alternative icons, but despite finding several discussions about a "restore defaults" icon, everyone's opinion seems to land on "there isn't a good icon for this."

What do you think about using text like "Reset" or "Restore default" ?

Reply:
Agree that refresh is not intuitive for reset.

Reply:
Here's a screenshot of what it looks like with the change:

<img width="672" alt="image" src="https://user-images.githubusercontent.com/1292638/188947120-4d59627f-1acc-409a-8c44-2a8707306a28.png">

I'll run this as well as Mark's comments by UX to make sure everyone is on board and then I can push a quick update. In addition to the icon looking almost identical to the browser "refresh" icon, this would change the UI from a single word "Options", to two new icons. Having one icon button and one text button is likely a smoother transition.

Reply:
There is also the undo icon which is the reverse of the screenshots above, and seems a little more semantically appropriate.

"Indicates the ability to undo a step in a historical log"

This isn't actually a "historical log", but it does undo the step of the user modifying the widget options.

<img width="672" alt="image" src="https://user-images.githubusercontent.com/1292638/188967918-1ab559da-b14b-494d-98df-4baa3653cd4c.png">